client/api/tag.js

```diff
@@ -0,0 +1,14 @@
+import { extractData } from './helpers';
+import request from './request';
+
+const urls = {
+  root: '/tags'
+};
+
+function getTags() {
```

Comment:
Since you already extracted this one consider moving repository related actions here (addTag / removeTag)